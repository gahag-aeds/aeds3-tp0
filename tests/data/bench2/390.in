390 195
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 
Add 239 368
Min 254 383
Sub 236 365
Sum 211 340
Add 217 346
Max 179 308
Add 75 204
Min 134 263
Add 19 148
Min 17 146
Add 1 130
Max 119 248
Sub 184 313
Sum 163 292
Add 69 198
Max 183 312
Sub 64 193
Sum 166 295
Sub 140 269
Max 27 156
Sub 210 339
Min 56 185
Sub 72 201
Max 106 235
Sub 2 131
Sum 77 206
Sub 166 295
Min 260 389
Add 23 152
Sum 118 247
Add 249 378
Max 1 130
Sub 244 373
Max 96 225
Add 83 212
Min 200 329
Add 15 144
Max 29 158
Add 205 334
Max 165 294
Add 177 306
Min 77 206
Add 23 152
Max 233 362
Add 239 368
Min 223 352
Add 155 284
Min 174 303
Sub 260 389
Sum 166 295
Sub 72 201
Sum 210 339
Add 221 350
Sum 143 272
Add 187 316
Max 223 352
Add 91 220
Min 92 221
Sub 94 223
Min 113 242
Add 81 210
Sum 82 211
Add 245 374
Max 64 193
Sub 50 179
Max 199 328
Add 3 132
Sum 64 193
Add 99 228
Max 80 209
Sub 100 229
Min 16 145
Sub 156 285
Max 254 383
Sub 248 377
Sum 7 136
Sub 216 345
Max 14 143
Sub 180 309
Max 215 344
Sub 52 181
Sum 252 381
Sub 36 165
Max 12 141
Sub 6 135
Sum 222 351
Sub 106 235
Sum 97 226
Add 53 182
Sum 199 328
Add 81 210
Min 134 263
Add 153 282
Max 198 327
Add 69 198
Sum 202 331
Sub 8 137
Max 72 201
Add 137 266
Max 107 236
Add 23 152
Min 236 365
Sub 122 251
Sum 50 179
Add 229 358
Min 241 370
Sub 56 185
Min 56 185
Sub 126 255
Max 236 365
Sub 142 271
Max 177 306
Add 99 228
Max 178 307
Add 189 318
Min 104 233
Add 139 268
Max 166 295
Sub 72 201
Max 64 193
Add 237 366
Sum 25 154
Add 69 198
Sum 129 258
Sub 94 223
Max 137 266
Sub 202 331
Max 101 230
Sub 80 209
Max 78 207
Add 207 336
Max 234 363
Add 53 182
Max 200 329
Sub 24 153
Min 153 282
Sub 180 309
Sum 79 208
Sub 208 337
Max 46 175
Sub 186 315
Sum 89 218
Sub 222 351
Sum 24 153
Sub 138 267
Min 22 151
Sub 260 389
Min 149 278
Sub 60 189
Sum 203 332
Sub 212 341
Sum 36 165
Add 99 228
Max 152 281
Sub 36 165
Min 192 321
Sub 160 289
Max 237 366
Add 33 162
Sum 112 241
Sub 186 315
Sum 111 240
Add 85 214
Max 238 367
Add 51 180
Max 108 237
Sub 2 131
Sum 230 359
Add 187 316
Sum 209 338
Add 147 276
Sum 112 241
Add 169 298
Sum 241 370
Sub 8 137
Max 47 176
Sub 134 263
Sum 139 268
Add 195 324
Min 193 322
Add 81 210
Min 18 147
Sub 100 229
Max 180 309
Add 41 170
Sum 135 264
Add 243 372
Min 72 201
Add 243 372
Max 15 144
Add 183 312
Max 168 297
Sub 258 387
Min 8 137
Add 17 146
