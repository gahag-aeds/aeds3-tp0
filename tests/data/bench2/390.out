1
6454
99
1
1
100
6795
98
6813
102
0
98
6530
-3
6465
101
97
-3
102
98
0
100
0
1
6849
6528
6886
101
1
0
6948
102
102
6764
100
3
102
6860
102
100
6321
101
6522
6350
6443
-1
100
6522
100
100
0
6592
1
0
101
99
99
-1
101
99
6478
7019
102
97
100
102
102
-1
6583
100
6486
6333
2
-3
6213
6111
96
-3
98
6149
6093
98
97
6225
6042
6395
6408
98
6462
-2
-1
97
6646
-2
99
100
-1
